src/effects/effectslot.cpp

```diff
@@ -147,10 +147,6 @@ void EffectSlot::loadEffect(EffectPointer pEffect) {
         m_pControlNumParameters->forceSet(pEffect->numKnobParameters());
         m_pControlNumButtonParameters->forceSet(pEffect->numButtonParameters());
 
-        // Enabled is a persistent property of the effect slot, not of the
-        // effect. Propagate the current setting to the effect.
-        pEffect->setEnabled(m_pControlEnabled->get() > 0.0);
-
```

Comment:
This is a no-op with the initial value of `m_pControlEnabled` equal to the initial value of Effect's `m_bEnabled`.